client/components/external-link/index.jsx

```diff
@@ -54,4 +35,22 @@ export default React.createClass( {
 			</a>
 		);
 	}
-} );
+}
+
+ExternalLink.defaultProps = {
+	iconSize: 18,
+	showIconFirst: false
+};
+
+ExternalLink.propTypes = {
```

Comment:
Minor note, but since we extend Component, we can inline defaultProps and propTypes. I tend to prefer this, since it's a bit easier to scan.

```jsx
class ExternalLink extends Component {
    static defaultProps = {
         //...
     };
     static propTypes = {
         //...
     };
//...
```